lib/datasource/conan/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`datasource/conan/index getReleases handles bad registries 1`] = `null`;
```

Comment:
Do not use snapshots for simple `null` values. Use a `toBeNull` matcher instead.